components/MatterInterface/DoneMatterEndpoint.h

```diff
@@ -5,29 +5,49 @@
 #include <esp_matter_core.h>
 #include <esp_matter_identify.h>
 
-//SEZ@Done: we define custum ID 
-#define ESP_MATTER_DONE_COFFEE_MAKER_DEVICE_TYPE_ID 0x1000
-#define ESP_MATTER_DONE_COFFEE_MAKER_DEVICE_TYPE_VERSION 1
+//SEZ@Done: we define custom ID 
+#define ESP_MATTER_DONE_TEST_DEVICE_TYPE_ID 0x1000
+#define ESP_MATTER_DONE_TEST_DEVICE_TYPE_VERSION 1
+
+#define ESP_MATTER_DONE_MASTER_POWER_KEY_DEVICE_TYPE_ID 0x1001
+#define ESP_MATTER_DONE_MASTER_POWER_KEY_DEVICE_TYPE_VERSION 1
+
+#define ESP_MATTER_DONE_MULTI_FUNCTION_SWITCH_DEVICE_TYPE_ID 0x1002
+#define ESP_MATTER_DONE_MULTI_FUNCTION_SWITCH_DEVICE_TYPE_VERSION 1
 
 namespace esp_matter {
 namespace endpoint {
-namespace done_coffee_maker {
+
+namespace done_MasterPowerKey {
+typedef struct config {
```

Comment:
rm empty line (20) and add one before typedef(22)

Reply:
it is like Namespace of Matter, done_coffe_maker already moved to a separate class.